Dockerfile

```diff
@@ -64,7 +106,7 @@ COPY --from=apt-dependencies /tmp/apt /
 ARG TARGETPLATFORM
 ARG CACHEID
 ARG PINOCCHIO_TAG=v2.6.9
-ARG PINOCCHIO_TESTS=ON
```

Comment:
Do you think this is okay or is it good practice to build tests?

Reply:
I think we shouldn't build tests personally. This is more for the developers of pinocchio than us. Unless pinocchio is so unstable that we need the tests to prove it's sane?

Reply:
No I don't think pinocchio is unstable, especially since we are using the same version for over 2 years now